docs/docs/validation-action.mdx

```diff
@@ -0,0 +1,246 @@
+---
+id: validation-action
+sidebar_label: Slot Validation Actions
+title: Slot Validation Actions
+abstract: Learn how `ValidationAction` class is implemented in the Rasa SDK.
+---
+
+There are two helper classes in Rasa SDK with the role of executing custom slot extraction and validation:
+- `ValidationAction`: the base class for custom actions extracting and validating slots that can be set or updated
+outside of a form context.
+- `FormValidationAction`: the base class for custom actions extracting and validating slots that are set only within
+the context of a form.
+
+In order to implement custom slot extraction and validation logic, you have the option of subclassing either
+`ValidationAction` or `FormValidationAction` class, depending on the context in which you prefer to set or update slots.
+
+## `ValidationAction` class
+
+You can extend the `ValidationAction` class in the Rasa SDK to define custom extraction and / or validation of slots that
+can be set or updated outside of a form context.
+
+:::note
+`ValidationAction` is intended for extracting slots **outside** the context of a form.
+It will ignore extraction and validation methods for any slots that have a form specified under the slot mapping's `conditions`.
```

Comment:
```suggestion
It will ignore extraction and validation methods for any slots that have a form specified under the [slot mapping's `conditions`](https://rasa.com/docs/rasa/next/domain#mapping-conditions).
```